### PR TITLE
WIP clusters: Beard communication layer

### PR DESCRIPTION
**Pull request by jacenkow**
* Adds a method to dispatch ```make_clusters``` using Celery task.

* Adds methods to query an Elasticsearch instance.

* Adds a method to create a new profile in ```hepnames / authors```.

* Adds a method to update ```recid``` (profile) for the given signature ```uuid```.

### CODE REVIEW

**Comment by jacenkow** [comment]
@kaplun

**Comment by jalavik** [comment]
I have some problems with the way the Celery and Elasticsearch instances are created. Can we use the existing instances provided by `invenio_celery.celery` and `invenio_ext.es`, as they are (1) already created for you, and (2) using the app configuration (like host URLs etc.).

(FYI: you left some potentially sensitive connection information in one of the files).

**Comment by jacenkow** [comment]
@jalavik thanks, forgot to ```git rm``` the old files :disappointed:

**Comment by jalavik** [comment]
@jacenkow Cool. Thanks.

**Comment by kaplun** [comment]
> And regarding celery, beard-server does not use the same RabbitMQ and the whole config file will be different.

Well that does not need to be necessary the case. I think we can simply configure the beard VMs to accept only beard celery tasks but to communicate still with the same rabbitmq instance...

File: invenio_beard/records/create.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    """
+
+    # Get author's name.
+    name = profile.get('name')
+
+    # Template of an initial record.
+    record = {'collections': [{'primary': 'HEPNAMES'}],
+              'name': {'value': name}}
+
+    # An author can have a main e-mail as well as an alternative one.
```
Comment (jacenkow) [inline invenio_beard/records/create.py]: Sorry, I read ```alternative_email``` instead of ```alternative_name``` from ```authors``` block.

**Pull request closed without merging**